src/tests/JIT/HardwareIntrinsics/Arm/Shared/SveLoadVectorMaskedTest.template

```diff
@@ -0,0 +1,270 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I think this file is doing the same job as `SveLoadMaskedUnOpTest.template`, which is being used for 
`Vector<byte> LoadVector(Vector<byte> mask, byte* address)` etc.

Your new file is better. Can you combine them both into a single template? (either naming is fine)

Reply:
+1

Reply:
Originally I wanted this, but it's a bit more difficult because the other Load APIs take different operand types. Can we do that one in another PR?